zone/attack.cpp

```diff
@@ -4314,7 +4314,12 @@ void Mob::HealDamage(uint64 amount, Mob* caster, uint16 spell_id)
 				FilteredMessageString(caster, Chat::NonMelee, FilterSpellDamage,
 					YOU_HEALED, caster->GetCleanName(), itoa(acthealed));
 
-				if (caster != this)
+				if (RuleB(Bots, Enabled) && caster->IsBot() && RuleB(Bots, DisplayHealDamage)) {
+					caster->CastToBot()->GetBotOwner()->FilteredMessageString(caster->CastToBot()->GetBotOwner(),
+						Chat::NonMelee, FilterSpellDamage, GENERIC_9_STRINGS,
+						caster->GetCleanName(), " healed ", this->GetCleanName(), " for ", itoa(acthealed), " hit points.", " ", " ", " ");
```

Comment:
Don’t have to use `this` here since it’s implied by scope.

Reply:
This is still being used

Reply:
This is used to allow a BotOwner to receive heal messages for their bots, when their bot heals someone else.  Normal messaging will display, for example and hot on the bot owner, though not if the bot heals player x.  This allows the bot owner to see that heal damage.